js/app/demo/app.jsx

```diff
@@ -63,27 +63,29 @@ define(['react', 'jsx!editor', 'jsx!messages', 'jsx!fixedCode', 'jsx!configurati
                 }
             }());
 
-            var initialState = urlState || storedState || {
-                options: {
-                    parserOptions: {
-                        ecmaVersion: 5,
-                        sourceType: 'script',
-                        ecmaFeatures: {}
+            var initialState = Object.assign(
+                { fix: false },
+                urlState || storedState || {
+                    options: {
+                        parserOptions: {
+                            ecmaVersion: 5,
+                            sourceType: 'script',
+                            ecmaFeatures: {}
+                        },
+                        rules: (function() {
+                            var result = {};
+                            rules.forEach(function(rule, ruleId) {
```

Comment:
nit: You can use reduce and not use variables at all.

Reply:
I agree, although this code was already here before -- this PR just modifies the indentation. I can make the fix to use reduce in a separate PR.

Reply:
Actually, it's nontrivial to do this because `rules` is a `Map`, not an array. So it would require something like

```js
rules: Array.from(rules).filter(function(ruleData) {
    return ruleData[1].meta.docs.recommended;
}).reduce(function(result, ruleData) {
    result[ruleData[0]] = 2;
    return result;
}, {})
```